packages/instrumentations/package.json

```diff
@@ -39,6 +39,7 @@
     "test": "swtest -p test/tsconfig.json -c src"
   },
   "dependencies": {
+    "@fastify/otel": "^0.4.0",
```

Comment:
Dropping Community instrumentor for native instrumentor? Yay

Reply:
...Maybe just update the PR title and description with that info, just in case there's a future historical dig.